### PR TITLE
Implemented Asset control, for cached and shared visual UX assets

### PR DESCRIPTION
**Pull request by Duckers**
This PR contains:
- [x] Changelog
- [x] Documentation
- [x] Tests

### CODE REVIEW

**Comment by Duckers** [comment]
Hold off merge, found one bug

**Comment by Duckers** [comment]
Bug fixed & test added

**Comment by Duckers** [comment]
More bugs discovered

**Comment by Duckers** [comment]
Works now

**Comment by Duckers** [comment]
Seems to still need new Uno, with this; https://github.com/fusetools/uno/pull/1193

File: Source/Fuse.Controls.Panels/Asset.uno [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+		}
+
+		string _hash;
+		void ComputeHash()
+		{
+			if (_hash != null) return;
+
+			var hash = ClassName;
+			for (var i = 0; i < NameTable.Properties.Count; i++)
+				hash += ";" + MakeHash(NameTable.Properties[i].GetAsObject());
```
Comment (nekronos) [inline Source/Fuse.Controls.Panels/Asset.uno]: Does this include properties like `IsEnabled`?
For example:
```
<Asset ux:Class="MyIcon">
	<Image ux:Name="image" />
	<WhileDisabled>
		<Change image.Color="#888" />
	</WhileDisabled>
</Asset>
```

**Comment by mortoray** [comment]
Closing as abandoned. Feature recorded in https://github.com/fusetools/fuselibs-public/issues/824

**Pull request closed without merging**